.github/workflows/build-ios.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: |
-            example/ios/Pods
+            package/example/ios/Pods
```

Comment:
Are you sure that didn't use `package/` as a default path already? I need to double check in the CI if that works today

Reply:
This wasn't updated when you moved the code to `package` directory, so I assume CI doesn't know that it should look for `package/`

Reply:
Yea but I thought that CI is running in the `package/` working directory, I might be wrong though...